doctrine.rst

```diff
@@ -340,6 +340,8 @@ and save it!
     namespace App\Controller;
 
     // ...
+    use Symfony\Component\HttpFoundation\Response;
```

Comment:
not sure it should be, as `AbstractController` is missing, too

Reply:
Hmm, the MakerBundle is used some lines above this to generate the controller. It already adds the abstract controller, but the response is not in there (See PR description). So I think it makes sense to add it explicitly in the docs (we usually add use statements in examples if they are new in that code block)

Reply:
Didn’t notice it 👍🏻